src/Knp/Menu/Util/MenuManipulator.php

```diff
@@ -211,18 +193,17 @@ public function toArray(ItemInterface $item, $depth = null)
      * The subItem can be one of the following forms
      *   * 'subItem'
      *   * ItemInterface object
-     *   * array('subItem' => '@homepage')
-     *   * array('subItem1', 'subItem2')
-     *   * array(array('label' => 'subItem1', 'url' => '@homepage'), array('label' => 'subItem2'))
+     *   * ['subItem' => '@homepage']
+     *   * ['subItem1', 'subItem2']
+     *   * [['label' => 'subItem1', 'url' => '@homepage'], ['label' => 'subItem2']]
      *
-     * @param ItemInterface $item
-     * @param mixed         $subItem A string or array to append onto the end of the array
+     * @param mixed $subItem A string or array to append onto the end of the array
      *
      * @return array
```

Comment:
this could be improved by adding a more precise type for phpstan in `@phpstan-return`:

```
@phpstan-return list<array{label: string, url: string|null, item: ItemInterface|null}>
```

Reply:
As noted in main PR comment, I plan to add all phpstan stuff eventually